community/resources/best-practices.md

```diff
@@ -0,0 +1,41 @@
+---
+group: community
+title: Community Contributed Best Practices
+---
+
+This section covers a broad scope of Community contributed best and leading practices, common solutions, and recommendations for Magento. 
+
+Topics and areas include:
+
+* Project Setup
+* Theme development (when and were to make a new theme)
+* Composer use (when to create a new repo)
+* Local environment options
+
+<div class="bs-callout bs-callout-info" markdown="1">
```

Comment:
Use the {:.bs-callout .bs-callout-info} structure instead a div

Reply:
Fixed, also fixed in sources.md